### PR TITLE
Look and feel (2 columns style)

### PR DESCRIPTION
**Pull request by defrance**
Just a look and feel update of this boring screen

### CODE REVIEW

File: htdocs/compta/paiement.php [hunk 1]
```diff
@@ -449,19 +450,22 @@ function callForResult(imgId)
 		}
 
 		print '<form id="payment_form" name="add_paiement" action="'.$_SERVER["PHP_SELF"].'" method="POST">';
-		print '<input type="hidden" name="token" value="'.newToken().'">';
+		print '<input type="idden" name="token" value="'.newToken().'">';
```
Comment (eldy) [inline htdocs/compta/paiement.php:453]: ???

File: htdocs/compta/paiement.php [hunk 2]
```diff
@@ -449,19 +450,22 @@ function callForResult(imgId)
...
-		print '<table class="border centpercent">';
+		print '<div class="fichecenter">';
+		print '<div class="fichehalfleft">';
+		print '<div class="underbanner clearboth"></div>';
+	
+		print '<table class="noborder tableforfield" centpercent>';
 
 		// Third party
-		print '<tr><td class="titlefieldcreate"><span class="fieldrequired">'.$langs->trans('Company').'</span></td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
+		print '<tr><td width=30% class="fieldrequired">'.$langs->trans('Company').'</td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
```
Comment (eldy) [inline htdocs/compta/paiement.php:468]: Use of percent is deprecated. You must use a css style line "titlefieldcreate" for example;

File: htdocs/compta/paiement.php [hunk 3]
```diff
@@ -456,12 +457,15 @@ function callForResult(imgId)
...
-		print '<table class="border centpercent">';
+		print '<div class="fichecenter">';
+		print '<div class="fichehalfleft">';
+		print '<div class="underbanner clearboth"></div>';
+
+		print '<table class="noborder tableforfield" centpercent>';
 
 		// Third party
-		print '<tr><td class="titlefieldcreate"><span class="fieldrequired">'.$langs->trans('Company').'</span></td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
+		print '<tr><td class="width25p fieldrequired">'.$langs->trans('Company').'</td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
```
Comment (eldy) [inline htdocs/compta/paiement.php:468]: Why replacing the css titlefieldcreate used by all forms to defined the width of first column ?

Comment (defrance) [reply htdocs/compta/paiement.php:468]: because the column is too small (we have carriage return on other fields line)

**Comment by altairis-tof** [comment]
hello , i think it would be great to systematically post screen images to help understand what we are talking about.

**Comment by eldy** [comment]
Even if this PR goal is to fix a bug, it seems it introduces more problems than it solves, so request is closed.

**Pull request closed without merging**